### DISCUSSION TITLE
why we are in trouble? shared hosting not supporting phalcon V4/V5

### DISCUSSION THREAD

**Discussion by Sosiashvili** [phalcon/phalcon#128] [General]
What to do if we are working on small projects and do not need dedicated servers?

The leading hosts I talked to Falcon are not supported on shared hostings ))

I think we should think about making our own servers or making a list of which hosting companies support Falcon...

Now, whatever hosting I have left in the past I can not develop versions of either PHP or falcon...

I wonder about your opinions too?

If one day the hosting is deleted or not supported at all Phalcon :)) what we can do?

I have my business completely dependent on the web...

I think they are fighting for the best...

**Comment by Jeckerson** [comment]
Hello. That's why there will be Phalcon v6 in pure PHP, not as extension for cases like that. Which can be installed via `composer`.

**Comment by lifewcody** [comment]
Hey there!

We offer phalcon shared hosting. Send me an email and we can set you up (crobinson@izt.tech)

**Comment by borisdelev** [comment]
... most of shared hostings can add Phalcon, just write to support.

**Comment by Sosiashvili** [comment]
I wrote a lot to A2hosting but did not support Phalcon...

I need level up PHP 7.3 but there are no falcon 4 nor falcon 5 :)))

they told me that won't be this one on shared hosting... 

So I go with Nodejs for backend

**Reply by borisdelev** [reply]
Strange... On every shared hosting in my country can add Phalcon
I have installed even 2 versions- x4 and x5 with PHP8.

Anyway, ave waiting Phalcon 6!

**Comment by echodreamz** [comment]
Mine told me the main issue was something with PSR extension causing conflicts with other frameworks/apps that use the PHP files version of PSR, not the extension. Though, I think this was fixed recently.
